vumi_http_retry/tests/test_redis.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+from twisted.trial.unittest import TestCase
+from twisted.internet.defer import inlineCallbacks
+
+from vumi_http_retry.tests.redis import create_client, zitems
+
+
+class TestRedis(TestCase):
+    @inlineCallbacks
+    def setUp(self):
+        self.redis = yield create_client()
+
+    @inlineCallbacks
+    def tearDown(self):
+        yield self.redis.delete('foo')
```

Comment:
Yup, the plan is to add everything under a set prefix here too. Didn't know one could use glob-like expressions for selecting multiple keys, will do that, thanks :)